File: Day 14/day14.py
def min_moves_to_stables(reindeer: list[int], stables: list[int]) -> int:
  movimientos = 0
  for reno in reindeer:
    movimientos_reno = [abs(establo - reno) for establo in stables]
    min_mov_reno = min(movimientos_reno)
    idx_min_mov = movimientos_reno.index(min_mov_reno)
    stables.pop(idx_min_mov)
    movimientos += min_mov_reno

  return movimientos

File: Day 14/day14.ts
function minMovesToStables(reeinder: number[], stables: number[]): number{
    let movimientos:number = 0;
    for (const reno of reeinder){
        const movimientosReno = stables.map(establo => Math.abs(establo - reno));
        const minMovReno = Math.min(...movimientosReno);
        const idxMinMov = movimientosReno.indexOf(minMovReno);
        stables.splice(idxMinMov, 1);
        movimientos += minMovReno;
    }

    return movimientos;
}